samples/tab-sso/nodejs/.vscode/launch.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Please help add extensions.json file, so that user will get recommended to install the TTk. Refer to https://github.com/formulahendry/teamsfx-migrator/blob/main/templates/tab/.vscode/extensions.json

Reply:
added